tests/full_node_test.cpp

```diff
@@ -1183,12 +1183,18 @@ TEST_F(FullNodeTest, receive_send_transaction) {
 
 TEST_F(FullNodeTest, detect_overlap_transactions) {
   auto node_cfgs = make_node_cfgs<20>(5);
-  auto node_1_genesis_bal = own_effective_genesis_bal(node_cfgs[0]);
   auto nodes = launch_nodes(node_cfgs);
+  std::unordered_map<addr_t, val_t> expected_balances;
+  const auto node_1_genesis_bal = own_effective_genesis_bal(node_cfgs[0]);
+  for (const auto &node : nodes) {
+    expected_balances[node->getAddress()] = own_effective_genesis_bal(node->getConfig());
+  }
+
+  expected_balances[nodes[0]->getAddress()] = own_effective_genesis_bal(nodes[0]->getConfig());
```

Comment:
Maybe it could be replaced with just 
`auto expected_balances = effective_genesis_balances(node_cfgs[0])`

Reply:
But configs for all nodes should be the same. Other way it won't work. 
`effective_genesis_balances` is other method used by `own_effective_genesis_bal` and it is returning the same unordered_map type

Reply:
ooo, I see we have function that will do the same :D sorry